src/main/java/com/booster/server/VocabularyEntryController.java

```diff
@@ -29,4 +25,14 @@ VocabularyEntryDto create(@RequestBody CreateVocabularyEntryInput input) {
         return vocabularyEntryDto;
     }
 
+    @GetMapping(value = "/list", produces = APPLICATION_JSON_VALUE)
+    @ResponseStatus(HttpStatus.OK)
```

Comment:
`@ResponseStatus(HttpStatus.OK)` is redundant as a successful request has an `OK` status by default